### DISCUSSION TITLE
Line comments cached locally?

### DISCUSSION THREAD

**Discussion by joshuaguerin** [community/community#21845] [New to GitHub]
<p>I am an educator, and would like to use the web interface for assignment feedback, but would like a convienient way to store that feedback locally for my own records (e.g., accreditation purposes).</p>
<p>When you view files online you have the option to comment a particular line (the + button that appears). Are those cached locally in your repo in a way that is easy to access and associate with a particular file, or are those strictly a feature of <a href="http://github.com" rel="nofollow noopener">github.com</a>’s web interface?</p>
<p>Thanks!</p>
<p>Joshua</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/joshuaguerin">@joshuaguerin</a> </p>
<p>I’m afraid that those are a feature of GitHub and not stored locally. However, you can use the GitHub api to retrieve those comments.</p>

**Comment by joshuaguerin** [comment]
<p><a class="mention" href="/seveas">@seveas</a> </p>
<p>Thanks for the reply!</p>
<p>Joshua</p>
